charms/slurmctld/requirements.txt

```diff
@@ -1,2 +1,4 @@
 ops==2.17.1
 slurmutils<1.0.0,>=0.11.0
+influxdb==5.3.2
+netifaces-plus==0.12.4
```

Comment:
[foresight]: eventually we should think about replacing `netifaces` given that the original was deprecated and this fork is maintained by one person seemingly.

`netplan` ended up just reimplementing what they specifically needed in their own _utils.py_ module, but we can solve this later: https://github.com/canonical/netplan/pull/503